beps/bep_0048.rst

```diff
@@ -0,0 +1,81 @@
+:BEP: 48
+:Title: Tracker Protocol Extension: Scrape
+:Version: $Revision$
+:Last-Modified: $Date$
+:Author:  Jimmy Zelinskie <jimmyzelinskie@gmail.com>
+:Status:  Draft
+:Type:    Standards Track
+:Content-Type: text/x-rst
+:Created: 08-Sep-2016
+:Post-History:
+
+
+Abstract
+--------
+
+The original BitTorrent Protocol Specification (BEP 003) [#BEP_003]_ defines an exchange between a client and a tracker referred to as an Announce.
+An Announce has two objectives: update a peer's presence in the Swarm and discover other peers in the Swarm.
+As a result of dealing with multiple objectives, this exchange is computationally expensive when exercised at scale.
+In order to prevent trackers from becoming overloaded, Announces are specified to be sent on an interval dictated by the tracker's Announce response or when a client event occurs.
+However, in order to build responsive user interfaces, clients desired a way to query metadata about Swarms in bulk in the least computationally expensive way.
+The exchange that fetches this metadata for the clients is referred to as a Scrape.
+
+
+Implementation
+--------------
+
+As the UDP Tracker Protocol Specification (BEP 015) [#BEP_015]_ already defines the implementation of Scrape using the UDP protocol, the following will document the implementation for the HTTP Protocol.
+
+Changes to Announce
+...................
+
+The URL at which the Announce endpoint lives is now expected to contain the word ``announce`` somewhere on its path.
```

Comment:
>I was suggesting that this BEP does not restrict what URLs are valid announce urls. Doing so seems both unnecessary and somewhat futile. People are (I'm sure) already using announce URLs that don't satisfy this requirement. I don't see any benefit in breaking those.

I framed this as an extension to the original protocol. If it truly modified it, we should go back and edit BEP003. However, if you want to support scraping, you need to have `announce` _somewhere_ in the URL path so that we can replace it with scrape. I should probably make this more clear. 

>Instead, I would suggest you restrict the scope of this BEP to only apply to URLs that end with /announce (I think you want the slash too)

`/anounce` doesn't necessarily handle legacy trackers written in php that have paths ending with `/announce.php`.

I checked to see [what libtorrent does](https://github.com/arvidn/libtorrent/blob/578e9b35587213cbaa11565cc81dcf88d4b2ef4d/src/http_tracker_connection.cpp#L85) and it looks like exactly what I've described.